autoware_sensing_msgs/msg/SourcePointCloudInfo.msg

```diff
@@ -0,0 +1,13 @@
+# This message holds metadata information about a source point cloud segment
+# within a concatenated point cloud, including its spatial boundaries and structural properties.
+
+# Time of sensor data acquisition, and the coordinate frame ID (for 3d points).
+std_msgs/Header header
+
+string topic  # Source topic
+
+# The starting index (inclusive) of the point cloud segment within the concatenated point cloud
+uint32 idx_begin
+
+# The ending index (exclusive) of the point cloud segment within the concatenated point cloud
+uint32 idx_end
```

Comment:
I'm a little bit concerned on how this could be interpreted.

`idx_end` should mean the index of the ending point so it should be inclusive.

What are your thoughts on this? Are there prominent examples where this convention is followed?